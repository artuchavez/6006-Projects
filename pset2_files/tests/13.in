gradebook-small
3
A,B,C,D,E
A,B,C,D,E,A,D
1,1,1,1,1,1,1
3,2.2,2.5,2.8,2.1,1,1

