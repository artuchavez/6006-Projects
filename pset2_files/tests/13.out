A,B,C,D,E
2,2.2,2.5,1.9,2.1
2,2.1,2.2